packages/build/src/core/normalize_flags.ts

```diff
@@ -95,7 +97,14 @@ const getDefaultFlags = function ({ env: envOpt = {} }, combinedEnv) {
     testOpts: {},
     featureFlags: DEFAULT_FEATURE_FLAGS,
     statsd: { port: DEFAULT_STATSD_PORT },
-    tracing: { enabled: false, port: DEFAULT_OTEL_TRACING_PORT },
+    // tracing.apiKey defaults to '-' else we'll get warning logs if not using
+    // honeycomb directly - https://github.com/honeycombio/honeycomb-opentelemetry-node/issues/201
```

Comment:
Honeycomb emits a set of `console.warn` logs we can't skip if we don't provide an `apiKey` - https://github.com/honeycombio/honeycomb-opentelemetry-node/issues/201. This is why for production we pass `-`, since we're going to use a collector.